test/any/FetchSpecRunner-test.js

```diff
@@ -0,0 +1,186 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+describe('FetchSpecRunner', function() {
+  var FetchSpecRunner;
+  var PipelineMessage;
+  var ResultBox;
+  var runner;
+
+  beforeEach(function() {
+    foam.CLASS({
+      package: 'org.chromium.webidl.test',
+      name: 'MockHTTPRequest',
```

Comment:
I'm pretty sure I've seen this class somewhere else...

https://github.com/GoogleChrome/webidl-diff/blob/5cf5f808cac1324b5dd3362186a8d46a1e549c71/test/any/webIDLFileClasses-test.js#L23
https://github.com/GoogleChrome/webidl-diff/blob/385a102e07c0e9c3faa437df66951e6030567e4a/test/any/routes-helper.js#L27